config/jobs/kubernetes/sig-testing/files-remake.yaml

```diff
@@ -3,7 +3,9 @@ presubmits:
   - name: pull-kubernetes-files-remake
     path_alias: "k8s.io/kubernetes"
     decorate: true
-    always_run: true
+    optional: true
```

Comment:
`run_if_changed: foo` + `optional: false` is well behaved (required to pass for merge if it runs, considered OK if it's skipped), I suspect that's the behavior we actually want?